src/models/GOKU.jl

```diff
@@ -5,8 +5,21 @@
 
 struct GOKU <: LatentDE end
 
+@doc raw"""
+    apply_feature_extractor(encoder, x)
+Converts a batch of the initial high-dimensional data into lower-dimensional data (i.e. extracts features)
+# Arguments
+encoder: Has type Encoder{GOKU} or Encoder{LatentODE}\
+x: Initial data, has size (seq_len,), x[1] has size (input_dim, batch_size)
+"""
 apply_feature_extractor(encoder::Encoder{GOKU}, x) = encoder.feature_extractor.(x)
```

Comment:
Please, add the type declaration for x:   `x::Vector{Matrix}`

Reply:
And the same for LatentODE's method

Reply:
Yup, done